docs/data/material/components/alert/alert.md

```diff
@@ -8,81 +8,155 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/alert/
 
 # Alert
 
-<p class="description">An alert displays a short, important message in a way that attracts the user's attention without interrupting the user's task.</p>
+<p class="description">Alerts display brief messages for the user without interrupting their use of the app.</p>
 
-**Note:** This component is not documented in the [Material Design guidelines](https://m2.material.io/), but MUI supports it.
+## Introduction
+
+The Alert component can be used to provide important and potentially time-sensitive information in a way that does not interfere with the user's tasks. (Source: [ARIA APG](https://www.w3.org/WAI/ARIA/apg/patterns/alert/).)
+
+This component is not documented in the [Material Design guidelines](https://material.io/).
+
+:::warning
+Alerts should not be confused with alert _dialogs_ ([ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/alertdialog/)), which _are_ intended to interrupt the user to obtain a response.
+Use the Material UI [Dialog](https://mui.com/material-ui/react-dialog/) if you need this behavior.
+:::
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
-## Basic alerts
+## Usage
 
-The alert offers four severity levels that set a distinctive icon and color.
+After [installation](/material/getting-started/installation/), you can start building with this component using the following basic elements:
 
-{{"demo": "BasicAlerts.js"}}
+```jsx
+import Alert from '@mui/material/Alert';
 
-## Description
+export default function MyApp() {
+  return <Alert />;
+}
+```
 
-You can use the `AlertTitle` component to display a formatted title above the content.
+## Basics
 
-{{"demo": "DescriptionAlerts.js"}}
+The Alert component wraps around its content, and stretches to fill its enclosing container.
 
-## Actions
+The `severity` prop accepts four values representing different states—`success` (the default), `info`, `warning`, and `error`–with corresponding icon and color combinations for each:
 
-An alert can have an action, such as a close or undo button.
-It is rendered after the message, at the end of the alert.
+{{"demo": "BasicAlerts.js"}}
 
-If an `onClose` callback is provided and no `action` prop is set, a close icon is displayed. The `action` prop can be used to provide an alternative action, for example using a Button or IconButton.
+## Anatomy
 
-{{"demo": "ActionAlerts.js"}}
+The Alert component is composed of a root [Paper](/material-ui/react-paper/) component (which renders as a `<div>`) that houses an icon, a message, and an optional [action](#actions):
 
-### Transition
+```html
+<div
+  class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiAlert-root MuiAlert-standardSuccess MuiAlert-standard"
+  role="alert"
+>
```

Comment:
**Off-topic**

I don't recall what's the current state of the reflection on the non-slot class names (I don't know if there is a clear GitHub issue about it). If the direction is to remove non-slot class names in v6, then it will like this:

```suggestion
<div class="MuiPaper-root MuiAlert-root" role="alert">
```

It would make the DOM feel lighter, at the expense of making it harder to customize the component with CSS Modules or SAAS like it's currently possible with Bootstrap https://getbootstrap.com/docs/5.2/components/alerts/ or Vuetify. https://vuetifyjs.com/en/components/alerts/. These people would need to use the `classes` API.

Another option is https://twitter.com/devongovett/status/1576635415024390144